statuspage/src/store/index.js

```diff
@@ -135,7 +135,11 @@ export default new Vuex.Store({
                     group[check.namespace][groupBy] = {
                         type: check.type,
                         namespace: check.namespace,
-                        name: check.description ? check.description : check.name,
```

Comment:
Without e2e tests, would like a view on whether we want to maintain this fallback from whomever knows best.